src/window.rs

```diff
@@ -61,8 +61,10 @@ pub struct Window {
     font: Option<orbfont::Font>,
     pub widgets: RefCell<Vec<Arc<Widget>>>,
     pub widget_focus: Cell<usize>,
-    pub bg: Color,
-    pub running: Cell<bool>
+    pub bg: Cell<Color>,
+    pub running: Cell<bool>,
+    events: Vec<Event>,
```

Comment:
I'd favor internal mutability `RefCell<Vec<Event>>` over requiring windows to be changed to mutable.